src/files/types.ts

```diff
@@ -0,0 +1,56 @@
+/**
+ * Parameters for querying files in the Strapi upload plugin.
+ */
+export interface FileQueryParams {
+  /**
+   * Filters to apply to the query.
+   * Example: { name: { $contains: 'example' } }
+   */
+  filters?: Record<string, any>;
+
+  /**
+   * Sorting criteria for the query.
+   * Example: ['name:asc'] or 'name:asc'
+   */
+  sort?: string | string[];
+
+  /**
+   * Note: Pagination is not supported by the Strapi upload plugin API.
```

Comment:
Could we move this bit to the main jsdoc comment for the interface so that it shows up when hovering the type?